src/Forms/DatabaseFormFactory.php

```diff
@@ -370,12 +370,12 @@ public function createSelectUsers($name, $gibbonSchoolYearID = false, $params =
 
         if ($params['includeStaff'] == true) {
             $data = array('gibbonSchoolYearID' => $gibbonSchoolYearID, 'date' => date('Y-m-d'));
-            $sql = "SELECT gibbonPerson.gibbonPersonID, preferredName, surname, username 
+            $sql = "SELECT gibbonPerson.gibbonPersonID, preferredName, surname, username
                     FROM gibbonPerson
                     JOIN gibbonStaff ON (gibbonPerson.gibbonPersonID=gibbonStaff.gibbonPersonID)
                     WHERE gibbonPerson.status='Full'
                     ORDER BY gibbonPerson.surname, gibbonPerson.preferredName";
-            $result = $this->pdo->executeQuery($data, $sql);
+            $result = $this->pdo->executeQuery([], $sql);
```

Comment:
```suggestion
            $result = $this->pdo->select($sql);
```
Small tweak. `executeQuery` is the old method. We can use the newer `select`, which has data as a second optional param